packages/docusaurus-theme-search-algolia/src/validateThemeConfig.ts

```diff
@@ -18,6 +18,7 @@ export const DEFAULT_CONFIG = {
   appId: 'BH4D9OD16A',
 
   searchParameters: {},
+  searchPage: 'search',
```

Comment:
Let's rename this to `searchPagePath` and make it accept `string | null`. We have a similar API for the [`archiveBasePath`](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-blog#archiveBasePath). I think it's also worth supporting `false` for both APIs, but we should prefer consistency first.